app/controllers/hosts_controller.rb

```diff
@@ -96,13 +97,6 @@ def edit
   def update
     forward_url_options
     Taxonomy.no_taxonomy_scope do
-      # remove from hash :root_pass and bmc :password if blank?
```

Comment:
This part is not strictly related to the pull request. Just a small cleanup as that part of `update` is a bit hard to parse at first glance.